.dockerignore

```diff
@@ -1,8 +1,9 @@
 # project
 Dockerfile
 docker-compose.yml
-cla_frontend/settings/local.py
+cla_frontend/settings/local.py*
 static/
 node_modules/
 cla_frontend/assets-src/vendor/
 venv/
+reports/
```

Comment:
I think it would be worth adding `.git/` too.

Reply:
Good point